### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by anshuman7negi**
**this branch's core goals is to ;**

> Update values
> querying the table

link to my screenshot .
https://drive.google.com/drive/folders/1friBB6BkkyERxva2rVER20OwxZzwuluY?usp=sharing

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
 
-SELECT * from animals where name like '%mon';
-SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
-SELECT * FROM animals WHERE neutered = TRUE AND escape_attempt < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempt FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = TRUE;
-SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+delete from animals where date_of_birth > '2022-01-01';
```
Comment (binhussen) [inline queries.sql:37]: - kindly, Inside a transaction update the animals table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction as specified in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) 👍 
``` 
  BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

File: queries.sql [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+DELETE 1
```
Comment (binhussen) [inline queries.sql:38]: - Kindly, Inside a transaction:
     - Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
     - Update the animals table by setting the species column to pokemon for all animals that don't have species already set as specified in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) 👍
     
```
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

File: queries.sql [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+
```
Comment (binhussen) [inline queries.sql:39]: - kindly, Inside a transaction delete all records in the animals table, then roll back the transaction as specified in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) 👍
```
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;

```

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @anshuman7negi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Added Query with Transaction ✔️
- Well structured Code ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @anshuman7negi 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Transactions are implemented
-  ✅ Great work using the right GitFlow 
-  ✅ Successful queries 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**